t.rast.ndvi/t.rast.ndvi.py

```diff
@@ -129,26 +130,27 @@ def main():
         new_inputs.append("%s.%s@%s" % (strds, red_band, mapset))
         new_inputs.append("%s.%s@%s" % (strds, nir_band, mapset))
 
-        expression=("float(%(instrds)s.%(nir)s@%(mapset)s - %(instrds)s.%(red)s@%(mapset)s) / "
-                    "(%(instrds)s.%(nir)s@%(mapset)s + %(instrds)s.%(red)s@%(mapset)s)" %
+        expression = ("float(%(instrds)s.%(nir)s@%(mapset)s - %(instrds)s.%(red)s@%(mapset)s) / "
```

Comment:
Can't NIR and RED both be 0 in any of the datasets used, and thus NaN come out for the NDVI?

Reply:
In theory yes, but at least for Landsat and Sentinel-2, red and nir should always be > 0. Metadata for Sentinel-2 read
```
                <SPECIAL_VALUE_TEXT>NODATA</SPECIAL_VALUE_TEXT>
                <SPECIAL_VALUE_INDEX>0</SPECIAL_VALUE_INDEX>
```